projects/llm-d/visualizations/llmd_inference/plotting/throughput_comparisons.py

```
@@ -344,17 +346,44 @@ def filter_flavors(setting_lists, flavor_filter):
 
-            for flavor in simple_flavors:
-                header.append(html.H4(f"🔧 {flavor}"))
-                if flavor == "simple-tp4-x4":
-                    header.append(html.I(f"Skipped, not relevant for baseline."))
-                    continue
-
-                # Set llama3.3-70b model and specific simple flavor
-                baseline_settings = {"model": "llama3.3-70b", "load_shape": load_shape, "flavor": flavor}
+            # If only one platform, show all simple flavors together
+            if "platform" not in variables:
+                header.append(html.H4(f"🔧 Simple Flavor Comparison"))
```

Ruff (0.15.14)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)